package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "strip-debug",
-	"version": "5.0.0",
+	"version": "6.0.0",
```

Comment:
You should generally not bump a version in a PR. It's up to the maintainers to do this.